lib/mempool.go

```diff
@@ -1451,18 +1452,28 @@ func ComputeTransactionMetadata(txn *MsgDeSoTxn, utxoView *UtxoView, blockHash *
 		realTxMeta := txn.TxnMeta.(*NFTBidMetadata)
 		_ = realTxMeta
 
+		isBuyNow := false
+		// Check if the Bid is on a Buy Now NFT.
+		if val, exists := txn.ExtraData[IsBuyNowKey]; exists && reflect.DeepEqual(val, []byte{BoolToByte(true)}) {
```

Comment:
Hmmmm, since Rosetta depends on this, maybe we should block height restrict this extradata lookup? Just to be extra safe? This would align with the change I proposed in _connectUpdateNFT().